providers/tentacle.rb

```diff
@@ -172,6 +172,16 @@
     end
     new_resource.updated_by_last_action(actions_updated?([register_polling_instance]))
   else
+
+    firewall = windows_firewall_rule 'Octopus Deploy Tentacle' do
+      localport port.to_s
+      dir :in
+      protocol 'TCP'
+      firewall_action :allow
+      action :create
```

Comment:
Wait actually this should be in the configure action not the register action.  Sorry did not see this before. This issue is not just for the register action but in fact for the octopus deploy server to talk at all to the tentacle.